webtau-core/src/main/java/com/twosigma/webtau/expectation/equality/handlers/MapAndBeanCompareToHandler.java

```diff
@@ -55,10 +51,10 @@ public void compareEqualOnly(CompareToComparator comparator, ActualPath actualPa
         expectedMap.keySet().forEach(p -> {
             ActualPath propertyPath = actualPath.property(p);
 
-            if (! actualAsMap.containsKey(p)) {
-                comparator.reportMissing(this, propertyPath, expectedMap.get(p));
-            } else {
+            if (actualAsMap.containsKey(p)) {
```

Comment:
ouch, that looked like a real bug :(

Reply:
No, I just flipped the if statement.  I can't remember why though.  I think just because I felt it made more sense this way round 😄